packages/identifier/src/did.ts

```diff
@@ -38,7 +38,7 @@ export const ensureValidDid = (did: string): void => {
     throw new InvalidDidError('DID can not end with ":" or "%"')
   }
 
-  if (did.length > 8 * 1024) {
+  if (did.length > 2 * 1024) {
     throw new InvalidDidError('DID is far too long')
```

Comment:
since we've limited to 2, can probably just change the error text to "too long" instead of "far too long" 😅

Reply:
Since other errors return what's valid/invalid, shouldn't this error also mention the max length?